### PR TITLE
Adding test case for pod scenarios [OVN-control-plane]

### PR DESCRIPTION
**Pull request by shahsahil264**
@chaitanyaenr @paigerube14

### CODE REVIEW

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

File: ci-operator/step-registry/redhat-chaos/pod-scenarios/ovn-cp-disruption/redhat-chaos-pod-scenarios-ovn-cp-disruption-ref.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+    default: "openshift-ovn-kubernetes"
+  - name: POD_LABEL
+    default: "app=ovnkube-control-plane"
+  - name: DISRUPTION_COUNT
+    default: "1"
+  - name: EXPECTED_POD_COUNT
+    default: "3"
+  - name: KILL_TIMEOUT
+    default: "180"
+  - name: EXPECTED_RECOVERY_TIME
```
Comment (chaitanyaenr) [inline ci-operator/step-registry/redhat-chaos/pod-scenarios/ovn-cp-disruption/redhat-chaos-pod-scenarios-ovn-cp-disruption-ref.yaml:26]: @shahsahil264 is it captured from at least 10 manual runs? Thoughts?

Comment (shahsahil264) [reply ci-operator/step-registry/redhat-chaos/pod-scenarios/ovn-cp-disruption/redhat-chaos-pod-scenarios-ovn-cp-disruption-ref.yaml:26]: Hey @chaitanyaenr ,  not yet, I'll do the manual runs and change it!

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by paigerube14** [comment]
looks good!

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Comment by paigerube14** [comment]
Looks like we need to set an int here not float for the recovery time

```
2024-08-14 17:03:37,011 [INFO] scenario /tmp/pod_scenario.yaml
2024-08-14 17:03:37,411 [INFO] waiting 3.5 seconds for pod recovery, pod label selector: app=ovnkube-control-plane namespace pattern: openshift-ovn-kubernetes
2024-08-14 17:03:37,412 [ERROR] scenario exception: Validation failed for 'KillPodConfig -> krkn_pod_recovery_time': Must be an integer, float given
```

**Comment by shahsahil264** [comment]
/pj-rehearse periodic-ci-redhat-chaos-prow-scripts-main-4.16-nightly-krkn-hub-tests-416

**Pull request merged**